Grammar (#16276)
mdn/content: files/en-us/web/accessibility/aria/index.md

---
Accessible Rich Internet Applications **(<abbr>ARIA</abbr>)** is a set of [roles](/en-US/docs/Web/Accessibility/ARIA/Roles) and [attributes](/en-US/docs/Web/Accessibility/ARIA/Attributes) that define ways to make web content and web applications (especially those developed with JavaScript) more accessible to people with disabilities.

> It supplements HTML so that interactions and widgets commonly used in applications can be passed to assistive technologies when there is not otherwise a mechanism. For example, ARIA enables accessible JavaScript widgets to form hints and error messages, live content updates, and more.

Reviewer: The hints can be seen on non JS widget elements also.

I think it's a case of parallel structure gone wrong.
```suggestion
It supplements HTML so that interactions and widgets commonly used in applications can be passed to assistive technologies when there is no other mechanism. For example, ARIA enables accessible JavaScript widgets, hints and error messages, live content updates, and more.
```
Or
> For example, ARIA enables JavaScript widgets' accessibility, hints and error messages formation, live content updation, and more.

Reviewer 2: The original is a list of what ARIA enables. It enables accessible JS widgets. It enables accessible hints and error messages. It enables accessible live content updates. The original was correct. Closing.
